src/components/Input.jsx

```diff
@@ -0,0 +1,11 @@
+const Input = ({ placeholder, className, value, onChange }) => {
```

Comment:
💊 제안
공통 컴포넌트는 다양하게 사용될 수 있어야 합니다!
Input의 type 속성도 받을 수 있으면 좋을 것 같아요~

```suggestion
const Input = ({ type, placeholder, className, value, onChange }) => {
```